src/show/pages/public/CreateProject/Subviews/index.js

```diff
@@ -0,0 +1,443 @@
+    </div>
+  );
+};
+
+const ImagesStep = ({ nextStep, previousStep, project, setProject }) => {
+  return (
+    <div className='mt-12 px-20'>
+      <div className='text-center text-3xl text-gray-500'>
+        Enter your project photos URL
+      </div>
+      <div className='h-full mt-6'>
+        {project.images.map((image, index) => (
+          <TextField
+            key={index}
+            value={image}
+            className='rounded-2xl mt-2 items-center text-center'
+            onChange={e => {
+              let newImages = project.images;
+              newImages[index] = e.target.value;
+              setProject({ ...project, images: newImages });
+            }}
+          />
+        ))}
+      </div>
+      <div className='flex mt-14 justify-between cursor-pointer'>
+        <div
+          onClick={() => previousStep()}
+          className='flex items-center hover:scale-110 transition-all duration-300'>
+          <BsArrowBarLeft size='2rem' />
+          Back
+        </div>
+        <div
+          onClick={() => nextStep()}
+          className='flex items-center cursor-pointer hover:scale-110 transition-all duration-300'>
+          Next
+          <BsArrowBarRight size='2rem' />
+        </div>
+      </div>
+      <div className='mt-20 px-60'>
+        <ProgressBar percentage={80} color='bg-primary-600' />
+      </div>
+    </div>
+  );
+};
+
+const TeamMembersStep = ({ nextStep, previousStep, project, setProject }) => {
+  return (
+    <div className='mt-12 px-20'>
+      <div className='text-center text-3xl text-gray-500'>
+        Enter your project members
+      </div>
+      <div className='flex w-full mt-10'>
+        <div className='flex w-1/2 h-full'>
+          <div className='w-3/4 px-2'>
+            {project.team_members.map((member, index) => (
+              <TextField
+                key={index}
+                value={member.name}
+                className='rounded-2xl mt-2 items-center text-center'
+                onChange={e => {
+                  let newTeamMembers = project.team_members;
+                  newTeamMembers[index] = { ...member, name: e.target.value };
+                  setProject({ ...project, team_members: newTeamMembers });
+                }}
+              />
+            ))}
+          </div>
+        </div>
+        <div className='flex w-1/2 h-full justify-end'>
+          <div className='w-full px-2'>
+            {project.team_members.map((member, index) => (
+              <TextField
+                key={index}
+                value={member.avatar_url}
+                className='rounded-2xl mt-2 items-center text-center'
+                onChange={e => {
+                  let newTeamMembers = project.team_members;
+                  newTeamMembers[index] = {
+                    ...member,
+                    avatar_url: e.target.value
+                  };
+                  setProject({ ...project, team_members: newTeamMembers });
+                }}
```

Comment:
Move this logic into a separate function.

Reply:
roger that